main.go

```diff
@@ -44,12 +44,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	// IronIO
```

Comment:
same as above... potentially having it conditional would help to reduce endpoints exposed and unnecessary endpoints being exposed when not being used.